examples/4337-passkeys/src/logic/safe.ts

```diff
@@ -28,7 +28,7 @@ const SafeProxyBytecode =
 function getSignerAddressFromPubkeyCoords(x: string, y: string): string {
   const deploymentCode = ethers.solidityPacked(
     ['bytes', 'uint256', 'uint256', 'uint256'],
-    [SafeWebAuthSignerBytecode, x, y, P256_VERIFIER_ADDRESS],
+    [SafeWebAuthnSignerSingletonBytecode, x, y, P256_VERIFIER_ADDRESS],
```

Comment:
This seems wrong to me - shouldn't it use the proxy code for computing the `CREATE2` address? Might be worth waiting for #389 to rebase and test the example application works.

In general, I think we need to find a good way to make the DevEx in this repo better for incorporating changes to the `passkey` and `4337` code into the example application, because it is becoming a bit of a maintenance burden ATM.

Reply:
Also, I think it is missing the `singletonAddress` for computing the `CREATE2` address.

Reply:
Just FYI,  in https://github.com/safe-global/safe-modules/pull/389 I replaced this with an RPC call to the factory's `getSigner` function

Reply:
marking as resolved after rebase